File: src/scoreboard-server/scoreboard_server/services/api/admin/eval/start.py
from __future__ import annotations

from typing import Any

from scoreboard_server.cores.scheduler_admin import controller
from scoreboard_server.dtos.api.admin.eval.start import AdminEvalStartResponse


def admin_eval_start_response(payload: dict[str, Any]) -> AdminEvalStartResponse:
    return controller.start(payload)

File: src/scoreboard-client/lib/api/admin/eval/start.ts
import { postJsonAuth } from "../../../http";
import type { AdminEvalStartRequest, AdminEvalStartResponse } from "../../../dtos/api/admin/eval/start";

export function adminStart(payload: AdminEvalStartRequest): Promise<AdminEvalStartResponse> {
  return postJsonAuth<AdminEvalStartResponse>("/api/admin/eval/start", payload);
}
